src/main/java/com/meilisearch/sdk/SettingsHandler.java

```diff
@@ -475,6 +476,44 @@ TaskInfo resetPaginationSettings(String uid) throws MeilisearchException {
                 settingsPath(uid).addSubroute("pagination").getURL(), TaskInfo.class);
     }
 
+    /**
+     * Gets the faceting settings of the index
+     *
+     * @param uid Index identifier
+     * @return a Faceting instance that contains all faceting settings
+     * @throws MeilisearchException if an error occurs
+     */
+    Faceting getFacetingSettings(String uid) throws MeilisearchException {
+        return httpClient.get(settingsPath(uid).addSubroute("faceting").getURL(), Faceting.class);
+    }
+
+    /**
+     * Updates the pagination settings of the index
+     *
+     * @param uid Index identifier
+     * @param faceting a Faceting instance that contains the new faceting settings
+     * @return TaskInfo instance
+     * @throws MeilisearchException if an error occurs
+     */
+    TaskInfo updateFacetingSettings(String uid, Faceting faceting) throws MeilisearchException {
+        return httpClient.patch(
+                settingsPath(uid).addSubroute("faceting").getURL(),
+                faceting == null ? httpClient.jsonHandler.encode(faceting) : faceting,
+                TaskInfo.class);
+    }
+
+    /**
+     * Deletes the faceting settings of the index
```

Comment:
```suggestion
     * Reset the faceting settings of the index
```
It resets this parameter to its default value of 100